CMakeLists.txt

```diff
@@ -13,7 +13,7 @@ set(BUILD_SHARED_LIBS TRUE CACHE BOOL "Build shared libraries")
 ament_vendor(spdlog_vendor
   SATISFIED ${spdlog_FOUND}
   VCS_URL https://github.com/gabime/spdlog.git
-  VCS_VERSION v1.9.2
+  VCS_VERSION v1.12.0
```

Comment:
Any reason why the version on line 7 above wasn't bumped?

```cmake
find_package(spdlog 1.5.0 QUIET)
```

I'm assuming it was an oversight, since this vendor package first tries to use the system package if available https://github.com/ros2/spdlog_vendor/blob/233537fc15af24bc653fcbf7cad64e81a593cc36/package.xml#L9-L10 which should match the Ubuntu system package version, and otherwise builds it from source. I'm checking just in case, see: https://github.com/ros2/spdlog_vendor/pull/40#discussion_r2169409432